flow/ebpf.go

```diff
@@ -0,0 +1,290 @@
+package flow
+
+import (
+	"encoding/hex"
+	"net"
+	"strings"
+	"syscall"
+	"time"
+	"unsafe"
+
+	"github.com/skydive-project/skydive/common"
+)
+
+// #cgo CFLAGS: -I../probe/ebpf
+// #include "flow.h"
+import "C"
```

Comment:
this is breaking windows cross compiling target
generate a dummy implentation of EBFFlow in no_ebpf.go with header // +build !linux

Reply:
done